notebooks/spark/AerospikeSparkPython.ipynb

```diff
@@ -5,7 +5,7 @@
    "metadata": {},
```

Comment:
Remove these comments:

"Below, a seed address for your Aerospike database cluster is required

Check the given namespace is available, and your feature key is located as per AS_FEATURE_KEY_PATH

Finally, review https://www.aerospike.com/enterprise/download/connectors/ to ensure AEROSPIKE_SPARK_JAR_VERSION is correct"